pkg/subcmd/find.go

```diff
@@ -0,0 +1,100 @@
+package subcmd
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+
+	"github.com/Shizuoka-Univ-dev/cvpn/api"
+	"github.com/Shizuoka-Univ-dev/cvpn/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+var (
+	volumeName   string
+	namePattern  string
+	pathPattern  string
+	recursiveFlg bool
+)
+
+func NewFindCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "find",
+		Short: "find files or directories which matched pattern",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			config, err := config.LoadConfig()
+			if err != nil {
+				return err
+			}
+			client := api.NewClient()
+			if err := client.LoadCookiesOrLogin(config.Username, config.Password); err != nil {
+				return err
+			}
+			volumeID, err := api.GetVolumeIDFromName(volumeName)
+			if err != nil {
+				return err
+			}
+
+			matchedPathes, err := searchMatches(client, args[0], volumeID)
+			if err != nil {
+				return err
+			}
+
+			for _, path := range matchedPathes {
+				fmt.Println(path)
+			}
+
+			return nil
+		},
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) < 1 {
+				return errors.New("path was not set")
```

Comment:
*path* はオプションの `--path` と混同しやすそうだから、こんな感じに直すと良さそう
```suggestion
				return errors.New("{starting-directory} was not set")
```